### ISSUE TITLE
Re-think PRO features

### ISSUE THREAD

**Issue by ameshkov** [AdguardTeam/AdguardForiOS#445]
We may need to react on this:
https://9to5mac.com/2017/07/15/apple-reportedly-shifts-app-store-policy-to-stop-adblockers-outside-of-safari/

Here's what I suggest:

1. Rename the "System-wide ad blocking" into "Firewall"; (UPD: let's merge blacklist/whitelist/log into the DNS settings, that'd be easier).
2. By default, it should not use any filter except the manually entered "blacklist" and "whitelist". We should change the description accordingly;
3. We should change the app store description and screenshots and remove the text about "ad blocking in apps" from there.
4. We should update descriptions inside of the app and get rid of the in-app ad blocking mentions there.

@vbagirov please attach the updated strings and images to this task and not update it in oneskyapp yet. We'll do this only when we're sure that they reject the current app.

**Comment by planet0** [comment]
Wish Adguard all the best with this, as when I was reading this I immediately thought about Adguard Pro. I hope there will be a way around it so we can at least use Adguard DNS if filters are not accepted.

**Comment by ameshkov** [comment]
And here we go:
![itunes connect 2017-07-18 10-55-09](https://user-images.githubusercontent.com/5947035/28306357-ac580b58-6ba7-11e7-9356-0d35e17e9793.png)

**Comment by ameshkov** [comment]
@vbagirov here is a PSD with the app screenshots, it should also be updated:
https://www.dropbox.com/s/r386w5fg4feiv0e/app_store_screens.psd?dl=0

**Comment by planet0** [comment]
Oh no... 😞 
Good luck guys.

**Comment by ameshkov** [comment]
@IvanIin @vbagirov here is a mockup:
https://app.moqups.com/ameshkov/rbJNBa7sH3/view

Please polish it, but I guess you've got the idea

**Comment by ameshkov** [comment]
Guys, let's also remove AG DNS servers from the list of pre-defined.

People will be able to add them manually anyway.

**Comment by X8716e** [comment]
My initial ideas, some of which I hope are of use to you, are these:

1. Weblock, as mentioned in the above article, has been going strong for 5 years. That was the first app I ever paid for, if not the only. There was nothing like it at the time. Their team has always been responsive and friendly when contacted for help. Perhaps they'd be willing to brainstorm.

On another note, I was wondering why they revamped the design of the app when the old design was in fact more user-friendly and less resource hungry. I dislike eye-candy for its own sake. But apparently it's not 'just eye candy'; at least after reading the absurdly vague AppStore guidelines. 

2. While browsing other system wide adblockers in the AppStore, I noticed that another one has the added 'functionality' of keeping track of your saved data charges.

Though it doesn't quite go that far, as it only lists the amount of data you've saved, though a the addition of a single input field of the rates you're charged resulting in tye output of the total charges you do save may be something to consider. (It further allows people not to think, which is what I see it as, though Apple apparently may see it as useful functionality).

To recap: apparently Apple prefers style over substance, especially when it reduces peoples' need for independent thought despite what they may tell themselves and the world. So maybe by playing their game through using the above 2 methods, as bitter a pill it may be, just may be a temporary fix to continue Adguard Pro to evolve in the ways that matter.

**Comment by JayBrown** [comment]
Question: independent of Adguard Pro for iOS and any upcoming changes, can iOS users not use your Adguard DNS? https://adguard.com/en/adguard-dns/overview.html Home > Settings > WiFi > active network > DNS. I know it's not possible for mobile networks, but users would at least have the DNS option system-wide when using a WiFi connection, regardless of what happens to Adguard Pro.

**Comment by planet0** [comment]
That is true, however the ability to use Adguard DNS on mobile data is very important if system-wide filtering is removed, and it can get annoying for users who need to connect to multiple Wi-Fi networks.

**Comment by vbagirov** [comment]
@ameshkov @IvanIin new texts for appstore and relevant texts for UI
https://app.box.com/s/i4z4gh8p6ukgauhqhjzgtaecxszu5cxv

**Comment by ameshkov** [comment]
@vbagirov thank you! Please don't forget about App Store screenshots, there's still an in-app ad blocking mention there.

**Comment by vbagirov** [comment]
@ameshkov @IvanIin .psd is uploaded, same link.

**Comment by vbagirov** [comment]
Can we close the task or should we update the onesky translations first? @ameshkov

**Comment by ameshkov** [comment]
> Can we close the task or should we update the onesky translations first? @ameshkov

We'll close it once we pass the app store review

**Comment by ameshkov** [comment]
@X8716e 
> But apparently it's not 'just eye candy'; at least after reading the absurdly vague AppStore guidelines.

From what I read it seems that nothing will save Weblock, it will be removed from the AppStore. It should be extremely frustrating for the guys, they've just made a huge update and redesign, and now this :(

On the topic, at the moment I've no idea how it will end. If we are allowed to continue using VPN API for non-ad blocking features, there's a plenty of things we can do. The most obvious would be to alert you about tracking/ads in particular apps. Also, we'd like to implement DNSCrypt -- encrypted DNS protocol, which prevents third-parties from snooping on your browsing history.

**Comment by ameshkov** [comment]
1.3.0 passed the review, it "Pending Developer Release". It's late evening here so we will publish it tomorrow.

**Comment by numbergames** [comment]
Any chance you would consider not publishing the update?  Publish a new app instead so existing users can keep full functionality without having to ignore the update?

**Comment by ameshkov** [comment]
> Any chance you would consider not publishing the update? Publish a new app instead so existing users can keep full functionality without having to ignore the update?

There's a number of reasons why we'd better publish it:

1. Apple won't let the app stay in the AppStore and will remove it if it's not updated soon.
2. The app will eventually become incompatible with the filters on the server side.
3. The DNS settings functionality is still available so you can manually set it up to use [Adguard DNS](https://adguard.com/en/adguard-dns/overview.html).

**Comment by ameshkov** [comment]
https://blog.adguard.com/en/adguard-ios-1-3-0/
